update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "linux-x64-prod": {
+    "update": "http://dl.bleenco.io/bterm-0.7.2-x86_64.AppImage",
```

Comment:
can we use something like `-latest-` instead of specified version, so we don't need to specify version in that that file every time new version is published.